test/views/reliefs/avoidanceSchemes.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.reliefs
+
+import config.ApplicationConfig
+import forms.ReliefForms._
+import models.{IsTaxAvoidance, ReliefsTaxAvoidance, StandardAuthRetrievals}
```

Comment:
There's imports that aren't being used which can be removed

Reply:
imports optimised.